cities/queries/fips_query.py

```diff
@@ -74,26 +74,28 @@ def __init__(
         self.fips = fips
         self.lag = lag
         self.top = top
+        self.gdpVAR = "gdp"
```

Comment:
can you explain why use `self.gdpVAR` instead of a shorter string
`gdp`? Also, if you have a good reason and insist, rename as `self.gdp_var`.

Reply:
Ah, ok, I see your reason, you want this to apply to `gdp_ma` in the subclass. Still, can you rename please?:)